### PR TITLE
v0.8.x [firebase-walletconnect-push]

### PR DESCRIPTION
**Pull request by pedrouid**

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -28,12 +28,7 @@ exports.push = functions.https.onRequest(async function (req, res) {
     res.status(500).send('Error: sessionId is required')
   }
 
-  const dappName = req.body.dappName
-  if (!dappName) {
-    res.status(500).send('Error: dappName is required')
-  }
-
-  const body = 'New call request from ' + dappName
+  const body = 'New Request'
```
Comment (ligi) [inline functions/index.js:31]: the body is not needed - see #1

**Review by ligi — CHANGES_REQUESTED** [review]
the body is not needed - see #1

**Comment by pedrouid** [comment]
Closed. Changes introduced in v1.0.0-beta instead

https://github.com/WalletConnect-Labs/walletconnect-push

**Comment by ligi** [comment]
very strange - is there any document where I can read why 1.0.0 makes this obsolete?

**Comment by ligi** [comment]
Ah wait - mixed up the PRs - thought this is about the notification type.

**Comment by pedrouid** [comment]
All v0.8.x changes have been pushed to the v1.0.0-beta. 
All the new repos are temporarily listed here:
https://github.com/WalletConnect-Labs

**Pull request closed without merging**